Classes/Emogrifier.php

```diff
@@ -1046,6 +1079,20 @@ private function getOrCreateHeadElement(\DOMDocument $document)
     }
 
     /**
+     * Returns the existing body element in $document.
+     *
+     * @param \DOMDocument $document
+     *
+     * @return \DOMNode the body element
+     */
+    private function getBodyElement(\DOMDocument $document)
+    {
+        $body = $document->getElementsByTagName('body')->item(0);
```

Comment:
With "inlining" I mean "replace $body (where it's used) directly with its definition. :-)